source/particle/property/interface.cc

```diff
@@ -396,14 +396,15 @@ namespace aspect
 
 
       template <int dim>
-      std::vector<double>
-      Manager<dim>::initialize_late_particle (const Point<dim> &particle_location,
+      void
+      Manager<dim>::initialize_late_particle (typename ParticleHandler<dim>::particle_iterator &particle,
                                               const ParticleHandler<dim> &particle_handler,
```

Comment:
It would have been nice if we could ask the iterator for its `particle_handler`, but there is no such function :-(